packages/contracts/scripts/util/constants.ts

```diff
@@ -111,6 +113,69 @@ const ACTIVE_SHARED_RANDOMIZERS = {
   },
 };
 
+export function getActiveSharedSplitProvider(): string {
+  return "0x0000000004B100B47f061968a387c82702AFe946";
+}
+
+export function getActiveEngineImplementations(
```

Comment:
oops, I didn't realize I had these to update. instead of hard-coding these, can we make this function retrieve them from the active engine factory? (kind of like how we get active core registry from minter filter)

Reply:
Nice! Updated 🙏